app/src/main/java/com/endava/myendava/activities/SignInActivity.java

```diff
@@ -35,7 +45,17 @@
     @BindView(R.id.password)
     EditText mPasswordEditText;
 
+    @BindView(R.id.forgot_password_button)
+    Button mForgotPasswordButton;
+
+    @BindView(R.id.terms_and_cond_checkbox)
+    CheckBox mTermsAndCondCheckBox;
+
     private Unbinder mUnbinder;
+    private boolean isMailValid;
+
+
+    public static final String INVALID_EMAIL_MESSAGE = "Invalid email";
```

Comment:
constants should be on top of the file with just one extra space on top and bottom